Share/Share.Plugin.iOSUnified/ShareImplementation.cs

```
@@ -238,6 +247,54 @@ UIViewController GetVisibleViewController()
+                case ShareUIActivityType.CopyToPasteboard:
+                    return UIActivityType.CopyToPasteboard;
+                case ShareUIActivityType.Mail:
+                    return UIActivityType.Mail;
+                case ShareUIActivityType.Message:
+                    return UIActivityType.Message;
+                case ShareUIActivityType.PostToFacebook:
+                    return UIActivityType.PostToFacebook;
+                case ShareUIActivityType.PostToTwitter:
+                    return UIActivityType.PostToTwitter;
+                case ShareUIActivityType.PostToWeibo:
+                    return UIActivityType.PostToWeibo;
+                case ShareUIActivityType.Print:
+                    return UIActivityType.Print;
+                case ShareUIActivityType.SaveToCameraRoll:
+                    return UIActivityType.SaveToCameraRoll;
+
+                case ShareUIActivityType.AddToReadingList:
+                    return UIDevice.CurrentDevice.CheckSystemVersion(7, 0) ? UIActivityType.AddToReadingList : null;
+                case ShareUIActivityType.AirDrop:
+                    return UIDevice.CurrentDevice.CheckSystemVersion(7, 0) ? UIActivityType.AirDrop : null;
+                case ShareUIActivityType.PostToFlickr:
+                    return UIDevice.CurrentDevice.CheckSystemVersion(7, 0) ? UIActivityType.PostToFlickr : null;
+                case ShareUIActivityType.PostToTencentWeibo:
+                    return UIDevice.CurrentDevice.CheckSystemVersion(7, 0) ? UIActivityType.PostToTencentWeibo : null;
+                case ShareUIActivityType.PostToVimeo:
+                    return UIDevice.CurrentDevice.CheckSystemVersion(7, 0) ? UIActivityType.PostToVimeo : null;
+
+                case ShareUIActivityType.OpenInIBooks:
+                    return UIDevice.CurrentDevice.CheckSystemVersion(9, 0) ? UIActivityType.OpenInIBooks : null;
+
+                default:
+                    return null;
```

Should we throw an exception here? This should only happen when we add new members to the enum for upcoming iOS versions and forget to add them here